src/NuGet.Services.Staging.Manager/Controllers/StageController.cs

```diff
@@ -189,6 +188,15 @@ public IActionResult Query(Stage stage)
             return new JsonResult(searchResult);
         }
 
+        [AllowAnonymous]
+        [HttpGet("{id:guid}/autocomplete")]
+        [EnsureStageExists]
+        public IActionResult Autocomplete(Stage stage)
+        {
+            var autocompleteResult = _searchServiceFactory.GetSearchService(stage.Id).Autocomplete(Request.QueryString.Value);

```

Comment:
done

---
In reply to: [65841789](https://github.com/NuGet/NuGet.Staging/pull/58#discussion_r65841789) [](ancestors = 65841789)